```
FAILED sympy/testing/tests/test_code_quality.py::test_files - AssertionError: File contains trailing whitespace: /home/runner/work/sympy/sympy/sympy/utilities/tests/test_codegen.py, line 654.
```

--- a/sympy/utilities/tests/test_codegen.py
+++ b/sympy/utilities/tests/test_codegen.py
@@ -651,7 +651,6 @@ def test_ccode_unused_array_arg_func():
         '}\n'
     )
     assert source == expected
-    
 
 def test_empty_f_code():
     code_gen = FCodeGen()

